.github/workflows/test.yml

```diff
@@ -62,10 +62,18 @@ jobs:
           platform: linux-x64-gnu
       - name: Build
         shell: bash
-        env:
-          CARGO_BUILD_TARGET: ${{ steps.neon.outputs.target }}
```

Comment:
Removing this because 1) the default works fine and 2) this means that `cargo test` will share the same Cargo options. Sharing the same build options means that `cargo test` doesn't need to rebuild everything (which cuts Rust build times in CI in half).